src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -172,7 +191,7 @@ const HoloFuelDnaInterface = {
       const noDuplicateIds = _.uniqBy(listOfNonActionableTransactions, 'id')
       return noDuplicateIds.filter(tx => tx.status === 'pending').sort((a, b) => a.timestamp < b.timestamp ? -1 : 1)
     },
-    getPending: async transactionId => {
+    getPending: async (transactionId) => {
```

Comment:
My vote would be we don't wrap single params in parens in function literals.

so eg

`(firstParam, secondParam) => /* do stuff with firstParam and secondParam */`

but

`onlyParam => /* do stuff with onlyParam */``

What do you think about that convention?

Reply:
I vote that it should be in parens. 
I feel it makes it more readable. 
But I don't have any strong feelings about it.